webapp/install_mattermost_webapp.sh

```diff
@@ -0,0 +1,23 @@
+readonly COMMITHASH=e2a5293e2e1c99ba66c4ebd0e358a1daaad2c4ed
```

Comment:
Can we put this commit hash in `package.json` instead? We can read `package.json` here and access the field using `jq`.